website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -84,99 +84,63 @@ This page provides steps to install Appsmith on a Kubernetes cluster using the [
   helm show values appsmith-ee/appsmith  > values.yaml
   ```
-6. Run the below command to deploy Appsmith:
+5. Run the below command to deploy Appsmith:
 
   ```bash
   helm install appsmith-ee appsmith-ee/appsmith -n appsmith-ee --create-namespace -f values.yaml
   ```
-7. Get pod name with:
+6. Get pod name with:
 
   ```bash
   kubectl get pods -n appsmith-ee
   ```
  The above command displays the status of the pods. Proceed to the next step once the pod status is shown as _RUNNING_.
 
-8. To access and verify the installation locally, use the below command that forwards the port 8080 to port 80:
+7. To access and verify the installation locally, use the below command that forwards the port 8080 to port 80:
   
   ```bash
   kubectl --namespace appsmith-ee port-forward appsmith-ee-0 8080:80
   ```
-  In above command, `appsmith-ee-0` is the Appsmith pod name.
+  In the above command, `appsmith-ee-0` is the Appsmith pod name.
+
+8. Open [http://localhost:8080](http://localhost:8080) and wait for the server to come up. This can take up to 5 minutes. Once the server is up and running, you can access Appsmith at [http://localhost:8080](http://localhost:8080).
+
+9. Fill in your details to create an administrator account.
 
-9. Open [http://localhost:8080](http://localhost:8080) and wait for the server to come up. This can take up to 5 minutes. Once the server is up and running, you can access Appsmith at [http://localhost:8080](http://localhost:8080).
+10. Once you've created an account, you can either start with the free plan or activate your instance with a license key. If you want to generate a license key, sign up on [customer.appsmith.com](https://customer.appsmith.com) to create one, and then proceed to activate your instance using the newly generated license key.
 
-10. Log into your Appsmith account, and enter your license key to activate the instance.
+For high availability and scalability configuration, see the [Configure High Availability and Scalability](/getting-started/setup/installation-guides/kubernetes/configure-high-availability) guide. To expose Appsmith installation on the internet, see the [Expose K8s to Internet](/getting-started/setup/installation-guides/kubernetes/publish-appsmith-online) guide.
 
-For high availability and scalability configuration, see the [Configure High Availability and Scalability](/getting-started/setup/installation-guides/kubernetes/configure-high-availability) guide. To expose Appsmith installation on internet, see the [Expose K8s to Internet](/getting-started/setup/installation-guides/kubernetes/publish-appsmith-online) guide.
+## Install Appsmith Community
 
-</TabItem>
-</Tabs>   
+To install the Appsmith open source edition (Appsmith Community):
+
+1. Use the open source helm chart by running the below command which adds the Appsmith chart repository: 
+  ```bash
+  helm repo add appsmith https://helm.appsmith.com
+  ```
+2. Replace `appsmith-ee` with `appsmith` in the commands on this page.
```

Comment:
Quick check, is this correct? Here we have to replace `appsmith-ee` with `appsmith` is it, and not `appsmith-ce`?

Reply:
Yes, this is correct. It refers to the helm repository which is appsmith.